appcues/src/main/java/com/appcues/debugger/DebuggerStatusManager.kt

```diff
@@ -33,6 +33,7 @@ import kotlinx.coroutines.withContext
 import java.util.UUID
 import kotlin.time.Duration.Companion.seconds
 
+@Suppress("TooManyFunctions")
```

Comment:
Similar to the other - the need to pass `reset()` through here added a 15th function.